concent_api/core/tests/test_payments_sci_backend.py

```diff
@@ -280,3 +275,70 @@ def test_that_sci_backend_cover_additional_verification_cost_should_return_trans
             value=self.transaction_value,
             subtask_id=sci_backend._hexencode_uuid(self.task_to_compute.subtask_id),
         )
+
+    def test_that_handle_exception_if_sci_not_synchronized_raise_custom_exception_if_not_sync(self):
+
+        @handle_exception_if_sci_not_synchronized
+        def dummy_handle_exception_if_sci_not_synchronized(_sci_function):
```

Comment:
you don't need `_sci_function` here and `None` in the function call

Reply:
Done